attributes/default.rb

```diff
@@ -14,6 +14,8 @@
   pool_conf_dir = "/etc/php5/fpm/pool.d"
   if node.platform == "ubuntu" and node.platform_version.to_f <= 10.04
     conf_file = "/etc/php5/fpm/php5-fpm.conf"
+  elsif node.platform == 'ubuntu' and node.platform_version.to_f >= 16.04
+    conf_file = '/etc/php/7.0/fpm/php-fpm.conf'
```

Comment:
This change is spot on, but I feel the other attributes that mention `php5` should probably also be brought in line with this style.

As a result, this `if` statement should probably be expanded to incorporate those values, e.g. (online 14) `pool_conf_dir = "/etc/php5/fpm/pool.d"` should become `pool_conf_dir = "/etc/php/7.0/fpm/pool.d"`. Othwise we end up with references to both `php5` and `php/7.0`, which might be confusing.